problem-three.js

```diff
@@ -22,7 +22,23 @@ const mergeArrays = function (array1, array2){
     return resArray;
 }
 
+// more efficient solution
+const mergeArrays2 = function (array1, array2){
+    const resArray = [];
+
+    const longerArray = array1.length > array2.length ? array1 : array2;
+    const shorterArrayLength = longerArray === array1 ? array2.length : array1.length;
```

Comment:
you can just use the lengths to figure this out, doing is value comparison is more expensive

Reply:
I'm guessing you're talking about the shorterArrayLength? I can instead use Math.min() and pass it the lengths of both arrays to determine which one is shorter.